File: Kotlin/src/main/kotlin/d_while_loops/exercise_41.kt
package d_while_loops

// 41) Develop a program that displays the following count on the screen: 100 95 90 85 80 … 0 Finished!

fun main() {
    var counter = 100

    while (counter >= 0) {
        print("$counter.. ")
        counter-=5
    }
    print("Done!")
}

File: Dart/d_while_loops/exercise_41.dart
// 41) Develop a program that displays the following count on the screen:
// 100 95 90 85 80 … 0 Finished!

import 'dart:io';

void main() {
  List<int> vector = [];
  int counter = 100;
  int minSize = 0;
  int step = 5;

  while (counter >= minSize) {
    vector.add(counter);
    counter -= step;
  }

  for (int element in vector) {
    if (element > minSize) {
      stdout.write('$element, ');
    } else {
      stdout.write('$element. ');
    }
  }
  print('finished!');
}
